test/actions/addTokensToVault.test.ts

```diff
@@ -1,90 +1,86 @@
+    await airdrop(connection, payer.publicKey, 10);
 
-  describe('Positive scenario of adding tokens to vault', () => {
-    test('creation and adding of multiple mint tokens to newly created vault', async () => {
-      const TOKEN_AMOUNT = 2;
-      const externalPriceAccountData = await createExternalPriceAccount({ connection, wallet });
+    const TOKEN_AMOUNT = 2;
+    const externalPriceAccountData = await createExternalPriceAccount({ connection, wallet });
 
-      await pause(20000);
+    const { vault } = await createVault({
+      connection,
+      wallet,
+      ...externalPriceAccountData,
+    });
 
-      const { vault } = await createVault({
-        connection,
-        wallet,
-        ...externalPriceAccountData,
-      });
-
-      const testNfts = [];
-      const mintRent = await connection.getMinimumBalanceForRentExemption(MintLayout.span);
-      const accountRent = await connection.getMinimumBalanceForRentExemption(AccountLayout.span);
-
-      for (let i = 0; i < TOKEN_AMOUNT; i++) {
-        const mint = Keypair.generate();
-        const createMintTx = new CreateMint(
-          { feePayer: FEE_PAYER.publicKey },
-          {
-            newAccountPubkey: mint.publicKey,
-            lamports: mintRent,
-          },
-        );
+    const testNfts = [];
+    const mintRent = await connection.getMinimumBalanceForRentExemption(MintLayout.span);
+    const accountRent = await connection.getMinimumBalanceForRentExemption(AccountLayout.span);
 
-        const tokenAccount = Keypair.generate();
-        const createTokenAccountTx = new CreateTokenAccount(
-          { feePayer: FEE_PAYER.publicKey },
-          {
-            newAccountPubkey: tokenAccount.publicKey,
-            lamports: accountRent,
-            mint: mint.publicKey,
-          },
-        );
+    for (let i = 0; i < TOKEN_AMOUNT; i++) {
+      const mint = Keypair.generate();
+      const createMintTx = new CreateMint(
+        { feePayer: payer.publicKey },
+        {
+          newAccountPubkey: mint.publicKey,
+          lamports: mintRent,
+        },
+      );
 
-        const mintToTokenAccountTx = new MintTo(
-          { feePayer: FEE_PAYER.publicKey },
-          {
-            mint: mint.publicKey,
-            dest: tokenAccount.publicKey,
-            amount: 1,
-          },
-        );
+      const tokenAccount = Keypair.generate();
+      const createTokenAccountTx = new CreateTokenAccount(
+        { feePayer: payer.publicKey },
+        {
+          newAccountPubkey: tokenAccount.publicKey,
+          lamports: accountRent,
+          mint: mint.publicKey,
+        },
+      );
 
-        await sendAndConfirmTransaction(
-          connection,
-          Transaction.fromCombined([createMintTx, createTokenAccountTx, mintToTokenAccountTx]),
-          [FEE_PAYER, mint, tokenAccount, wallet.payer],
-          {
-            commitment: 'confirmed',
-          },
-        );
+      const mintToTokenAccountTx = new MintTo(
```

Comment:
I think we can shorten this code by using
https://github.com/metaplex-foundation/js/blob/main/src/actions/shared/mint.ts
Can you check it, please?

Reply:
Looks good, thanks